offline_imu/data_loading.hpp

```diff
@@ -0,0 +1,48 @@
+#include "common/csv_iterator.hpp"
+
+#include <eigen3/Eigen/Dense>
+#include <fstream>
+#include <map>
+#include <opencv2/core/mat.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/imgproc.hpp>
```

Comment:
It's a minor issue but I don't see why offline_imu needs opencv, so does LDFLAGS and CFLAGS in Makefile.